user/languages/r.md

```diff
@@ -0,0 +1,148 @@
+
+### Community-Supported Warning
+
+Travis CI support for R is contributed by the community and may be removed or
+altered at any time. If you run into any problems, please report them in the
+[Travis CI issue tracker](https://github.com/travis-ci/travis-ci/issues/new?labels=community:r)
+and cc [@craigcitro](https://github.com/craigcitro),
+[@eddelbuettel](https://github.com/eddelbuettel), and
+[@hadley](https://github.com/hadley).
+
+## Basic configuration
+
+If your R repository doesn't need any dependencies beyond those specified in
+your `DESCRIPTION` file, your `.travis.yml` can simply be
+
+    language: r
+
+## Configuration options
+
+Travis CI supports a number of configuration options for your R package.
+
+### Dependencies
+
+By default, Travis CI will install all R packages listed as dependencies in
+your package `DESCRIPTION` file. However, it's often necessary or desirable to
+install additional dependencies, or newer versions of those dependencies.
+Adding an entry to one of the lists below will install before building your
+package; note that these lists are processed in order, so entries can depend
+on dependencies in a previous list.
+
+* `apt_packages`: A list of packages to install via `apt-get`. Common examples
+  here include entries in `SystemRequirements` (such as curl or XML
+  libraries). This option is ignored on non-linux builds.
+
+* `brew_packages`: A list of packages to install via `brew`. This option is
+  ignored on non-OSX builds.
+
+* `r_binary_packages`: A list of R packages to install as binary packages on
+  linux builds, via Michael Rutter's
+  [cran2deb4ubuntu PPA](https://launchpad.net/~marutter/+archive/ubuntu/c2d4u).
+  These installs will be faster than source installs, but may not always be
+  the most recent version. Specify the name just as you would when installing
+  from CRAN. On OSX builds, these packages are installed from source.
+
+* `r_packages`: A list of R packages to install via `install.packages`.
+
+* `bioc_packages`: A list of [BioConductor](http://www.bioconductor.org/)
+  packages to install.
+
+* `r_github_packages`: A list of packages to install directly from github,
+  using `devtools::install_github` from the
+  [devtools package](https://github.com/hadley/devtools). The package names
+  here should be of the form `user/repo`.
+
+### Package check options
+
+You can use the following top-level options to control what options are used
+when building and checking your package:
+
+* `warnings_are_errors`: Set this option to `true` to force all warnings to
+  become failures. This is especially helpful when preparing your package for
+  submission to CRAN, and is recommended for most packages.
+
+* `r_build_args`: additional arguments to pass to `R CMD build`, as a single
+  string. Defaults to empty.
+
+* `r_check_args`: additional arguments to pass to `R CMD check`, as a single
+  string. Defaults to `--as-cran`.
+
+* `r_check_cran_incoming`: if `true`, sets the value of the
```

Comment:
isn't this still needed to pick up some checks? (i'm basing this solely on the [r internals](http://cran.r-project.org/doc/manuals/R-ints.html) manual, not any actual information.)